driver/xrt/include/fpgabuffer.hpp

```diff
@@ -19,20 +19,11 @@
 #pragma once
 #include "buffer.hpp"
 #include "common.hpp"
+#include <cstdlib>
```

Comment:
So cstdlib is needed for std::aligned_alloc? Why is it included here and not in common.cpp?